Zhukov/t7.py

```diff
@@ -0,0 +1,54 @@
+
+# написать функции:
+# get_max
+# get_min
+# get_mid
+# get_med без сортировки
+# get_metrics
+# для сортированных массивов
+
+def get_max(num_array):
+    max_value = None
+    for n in num_array:
+        if (max_value is None or n > max_value):
+            max_value = n
+    return(max_value)
```

Comment:
```suggestion
    return max_value
```
Применить ко всем файлам.
return - это не функция, а ключевое слово. Скобки в данном случае опциональны и опускаются